main.go

```diff
@@ -92,9 +95,27 @@ func main() {
 			args.WaitForDockerSeconds -= 5
 		}
 	}
+	
+	username := args.Username
+	password := args.Password
+	
+	if args.UsernameFILE != "" && args.PasswordFILE != "" {
+		contentUser, err := ioutil.ReadFile(args.UsernameFILE)
+		if err != nil {
+	  		log.Fatal(err)
+		}
+		username = string(contentUser)
```

Comment:
Do you think you should do https://pkg.go.dev/strings#TrimSpace in case someone puts a new line in the file? I am not too familiar with Docker secrets so not sure.